tutorials/mendix-onboarding/mendix-onboarding.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Get Started with SAP Cloud Platform Rapid Application Development by Mendix
-description: Set up your SAP Cloud Platform account so you can use SAP Cloud Platform Rapid Application Development by Mendix to build applications to be deployed on SAP Cloud Platform.
+title: Get Started with SAP Rapid Application Development by Mendix
+description: Set up your SAP BTP account so you can use SAP Rapid Application Development by Mendix to build applications to be deployed on SAP BTP.

```

Comment:
Please use the full name "Business Technology Platform" at least once before you use the abbreviation BTP.

Reply:
Done